### PR TITLE
dotCMS/core#22335  fix storybook tasks nx

### PR DESCRIPTION
**Pull request by oidacra**
Fix storybook apps.

### CODE REVIEW

**Review by hmoreras — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts [hunk 1]
```diff
@@ -17,11 +17,11 @@ export class DotContentletLockerService {
      */
     lock(inode: string): Observable<DotContentletLockResponse> {
         return this.coreWebService
-            .requestView({
+            .requestView<DotContentletLockResponse>({
                 method: 'PUT',
                 url: `/api/content/lock/inode/${inode}`
             })
-            .pipe(pluck('bodyJsonObject'));
+            .pipe(pluck('entity'));
```
Comment (hmoreras) [inline core-web/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts:24]: the response changed in the endpoint ?

File: core-web/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts [hunk 2]
```diff
@@ -33,10 +33,10 @@ export class DotContentletLockerService {
      */
     unlock(inode: string): Observable<DotContentletLockResponse> {
         return this.coreWebService
-            .requestView({
+            .requestView<DotContentletLockResponse>({
                 method: 'PUT',
                 url: `/api/content/unlock/inode/${inode}`
             })
-            .pipe(pluck('bodyJsonObject'));
+            .pipe(pluck('entity'));
```
Comment (hmoreras) [inline core-web/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts:40]: the response changed in the endpoint ?

File: core-web/apps/dotcms-ui/src/app/api/services/notifications-service.ts [hunk 3]
```diff
@@ -27,33 +27,33 @@ export class NotificationsService {
```
Comment (hmoreras) [inline core-web/apps/dotcms-ui/src/app/api/services/notifications-service.ts:27]: all the responses changed in the endpoints ?

File: core-web/libs/dotcms-js/src/lib/core/core-web.service.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 import { Subject, Observable, throwError } from 'rxjs';
-import { map, catchError, filter } from 'rxjs/operators';
+import { map, catchError, filter, tap } from 'rxjs/operators';
```
Comment (hmoreras) [inline core-web/libs/dotcms-js/src/lib/core/core-web.service.ts:3]: `tap` needed ?

**Review by hmoreras — CHANGES_REQUESTED** [review]

**Comment by oidacra** [comment]
Close in favor of https://github.com/dotCMS/core/pull/22514/

**Pull request closed without merging**